src/Compilers/Core/Portable/Symbols/ITypeSymbol.cs

```diff
@@ -111,6 +111,11 @@ public interface ITypeSymbol : INamespaceOrTypeSymbol
         /// has no concept of unmanaged types.
         /// </summary>
         bool IsUnmanagedType { get; }
+
+        /// <summary>
+        /// True if the type is a readonly struct.
```

Comment:
>a readonly struct [](start = 32, length = 17)

Consider changing to `True if the type is readonly.`

Reply:
Done.